omaha/core/scheduler_unittest.cc

```diff
@@ -0,0 +1,180 @@
+// Copyright 2007-2019 Google Inc.

```

Comment:
Copyright 2019 Google Inc.

Since this is a new file, that's what we do.

Reply:
Done